frontend/src/components/common/LabelListItem.tsx

```diff
@@ -6,7 +6,7 @@ export interface LabelListItemProps {
 }
 
 export default function LabelListItem(props: LabelListItemProps) {
-  const { labels } = props;
+  const { labels = [] } = props;
```

Comment:
This should be in a separate commit as it's an unrelated fix.

Reply:
Put it in the fix branch!